source/create_cert.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
I tend to use `#!/bin/bash`, interesting explanation https://unix.stackexchange.com/a/206366

Reply:
Ah this was Rider being 'helpful' ... I agree